lib/formatters/json-with-metadata.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @fileoverview JSON reporter
+ * @author Burak Yigit Kaya aka BYK
```

Comment:
I assume this was copy-pasted from the `json` formatter file, but feel free to replace this with your own name or username.